packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -2447,6 +2447,26 @@ type MessagesType = {
   'scenes.login.upgrade.2fa.item_1.text': 'Generate a second step of verification when you sign in.'
   'scenes.login.upgrade.2fa.item_2.title': 'YubiKey'
   'scenes.login.upgrade.2fa.item_2.text': 'Secure your wallet with a physical key or device.'
+  'scenes.login.upgrade.googleAuth.header': 'Set Up 2FA'
+  'scenes.login.upgrade.googleAuth.text': 'With your Google Authenticator app, scan the QR code below to make a secure connection.'
+  'scenes.login.upgrade.googleAuth.button': 'Next'
+  'scenes.login.upgrade.googleAuthVerify.header': 'Verify Your Google Account'
+  'scenes.login.upgrade.googleAuthVerify.text': 'Enter the 6-digit code you see in your Google Auth App.'
+  'scenes.login.upgrade.googleAuthVerify.6digitcode': '6 Digit Code'
+  'scenes.login.upgrade.googleAuthVerify.6digitcode.placeholder': 'Enter 6 digit code'
+  'scenes.login.upgrade.googleAuthVerify.button': 'Next'
+  'scenes.login.upgrade.yubikey.header': 'Verify with Your Yubikey'
+  'scenes.login.upgrade.yubikey.text': 'Insert the Yubikey into your computer´s USB port. Pair your Yubikey by tapping your key.'
+  'scenes.login.upgrade.yubikeyVerify.header': 'Yubikey Verified'
+  'scenes.login.upgrade.yubikeyVerify.text': 'Make sure your Yubikey is plugged in next time you log into your Blockchain account.'
+  'scenes.login.upgrade.yubikeyVerify.button': 'Next'
+  'scenes.login.upgrade.2faSuccess.header': 'You’re All Set!'
+  'scenes.login.upgrade.2faSuccess.text': 'Look for a confirmation email. You can now log in to the Wallet and Exchange with:'
+  'scenes.login.upgrade.2faSuccess.input_1': 'Your Email'
+  'scenes.login.upgrade.2faSuccess.input_2': 'New Password'
+  'scenes.login.upgrade.2faSuccess.input_3': 'New 2FA'
+  'scenes.login.upgrade.2faSuccess.footNote': 'You can now delete your old Exchange 2FA method.'
+  'scenes.login.upgrade.2faSuccess.button': 'Open Exchange'
```

Comment:
we dont want to repeat translations and for basic strings such as "next" we should have these already translated.  check `copy.next` or `button.next`.

Reply:
It seems for some labels that they aren't available

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/97299628/163809433-66745d69-6565-444f-b2ea-6c99921b2e14.png">

Reply:
Done, thanks!